### ISSUE TITLE
Fuel cell waste heat recovery issues

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6609]
Issue overview
--------------
User file with `Generator:Fuelcell:StackCooler` shows erratic heat recovery and generates copius repeated warnings of:
```
   ** Severe  ** Plant Component Generator:Fuelcell:StackCooler called "SOFC" was not found on any plant loops.
   **   ~~~   ** Component, type="Generator:Fuelcell:StackCooler", name="SOFC" was not found on any Branch.
```
And ultimately fails with
```
   ** Severe  ** Plant temperatures are getting far too cold, check controls and relative loads and capacities
   **   ~~~   **  During Warmup, Environment=CHICAGO OHARE INTL AP ANN CLG .4% CONDNS DB=>MWB, at Simulation time=07/21 03:00 - 03:10
   **   ~~~   ** PlantLoop Name (SupplySide)= HEATSYS1
```
[Hospital CHP-storage v8-9-FixVariableFlow-FixEMS-FixGas.idf.txt](https://github.com/NREL/EnergyPlus/files/1857716/Hospital.CHP-storage.v8-9-FixVariableFlow-FixEMS-FixGas.idf.txt)

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v8.9.0
 - Helpdesk ticket number 12516.

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by EnergyArchmage** [comment]
This is likely the same problem as in #6051.  The code for plant component types is all messed up between stack cooler and exhaust gas HX.  User file does not actually have a stack cooler but the code is looking for one.

**Comment by EnergyArchmage** [comment]
After fixing the issue behind #6051, the file still shows problems.  Found that the EMS code overriding generator dispatch was shutting down the fuel cell on design days.  
DISPATCH_FUELCELL,Line 1,SET FUELCELL_DISPATCH = FC_P*1000,0.0, 
The defect file has lots of complex EMS controls on plant equipment schemes and strange schedules that do not work right on design days.  The user has too many zero values in the controlling schedules for SummerDesignDay and WinterDesignDay.

By altering the EMS program DISPATCH_FUELCELL I was able to get the file to run and results look okay during the run period.   I don't believe there is any new issue here besides the duplicate of #6051.
